Heads Mindshit and credit inflation

DieSiege opened the pull request:
Майндщилд-то по сути убирает возможность промывать мозги. Не более. Он есть у офицеров, которые защищают станцию, но их нет у глав, которые этой станцией командуют? Huh. 
А еще во всём виноват Шкиппер, бейте его если хотите.
Ошибся с бранчами и случайно комитнул сюда. Изменил цены на различную еду.
<!--
Подробно про оформление ПРов можно прочитать тут: https://github.com/TauCetiStation/TauCetiClassic/wiki/Styling-of-Pull-Requests-for-Dummies
Там же написано про то, как сделать чейнджлог. 

!!!
Если авторство не полностью ваше, вы делаете порт с другого билда - укажите первоисточник изменений!
Для крупных комплексных изменений достаточно будет указать билд(ы)-первоисточник, в остальных случаях можете указать исходный ПР.
!!!

Для копипаста:
Список классификаторов для быстрого копирования: bugfix, rscadd, rscdel, image, sound, spellcheck, tweak, balance, map, performance, experiment.

Пример списка:
:cl:
 - image: Добавлен плакат с изображением статного мужчины с конусом на голове и арбузами вокруг него.
 - image: С плаката чужого в форме горничной убрана цензура.
-->
:cl:
 - balance: У глав теперь есть mindshield импланты с начала раунда.
 - balance: Немного увеличена цена еды и сигарет.

DieSiege committed:
Heads Mindshit

Дал главам майндщилд импланты

torn223 commented:
Но... Но... Недовольные властью главы... Предательства...

Skippeer12 commented:
Чтобы не вводить никого в заблуждение, это специальная мера чтобы избавится от генокрадов-ХоПов, тенелингов-СЕ, РД-культистов и СМО-трейторов, которым выпадали рольки за деклер реди в лобби.

TungstenOctopus commented:
Годно, правильно, потому что имплант лояльности было бы неудачно, а вот майндшилд для глав - мастхев и уместно

DieSiege committed:
Изменил цены в автоматах

Несколько изменил цены в автоматах. Сделал их больше. Инфляция кредитов :shrug:

DieSiege commented:
Короче, да. Тут кое-что добавилось. Посмотрите тоже.

WallShrabnic commented:
Для этого следовало сделать отдельный ПР ИМХО
